src/components/DeleteButton.js

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact'
+import { FaClose } from 'react-icons/lib/fa'
+import styled, { withTheme } from 'styled-components'
+import hoc from '../hoc'
+import { compose } from 'recompose'
+
+const BtnWrapper = styled.button`
```

Comment:
Why are we re-implementing the button? We should require our other button but provide default props like `color=red` + pass the close icon.

Reply:
The DeleteButton component is visually quite different, otherwise I would have done exactly as you said. This button will be used in modals and alerts.
I think that perhaps `CloseButton` might be a more accurate name.

<img width="136" alt="screen shot 2017-09-06 at 08 09 10" src="https://user-images.githubusercontent.com/15064535/30098650-be0b15ac-92da-11e7-9da4-1baf6b6a9750.png">